docs/Routes.md

```diff
@@ -5,18 +5,21 @@
 The routes methods will configure the endpoints of your application.
 You have two ways to declare a route with Fastify, the shorthand method and the full declaration.
 
-- [Full Declaration](#full-declaration)
-- [Route Options](#options)
-- [Shorthand Declaration](#shorthand-declaration)
-- [URL Parameters](#url-building)
-- [Use `async`/`await`](#async-await)
-- [Promise resolution](#promise-resolution)
-- [Route Prefixing](#route-prefixing)
-- Logs
+- [Routes](#routes)
+  - [Full declaration](#full-declaration)
+  - [Routes option](#routes-option)
+  - [Shorthand declaration](#shorthand-declaration)
+  - [Url building](#url-building)
+  - [Async Await](#async-await)
+  - [Promise resolution](#promise-resolution)
+  - [Route Prefixing](#route-prefixing)
+    - [Handling of / route inside prefixed plugins](#handling-of--route-inside-prefixed-plugins)
   - [Custom Log Level](#custom-log-level)
   - [Custom Log Serializer](#custom-log-serializer)
-- [Route handler configuration](#routes-config)
-- [Route constraints](#constraints)
+  - [Config](#config)
+  - [Constraints](#constraints)
+    - [Version Constraints](#version-constraints)
+    - [Host Constraints](#host-constraints)
```

Comment:
why was this changed?

Reply:
Good question. To be honest I don't understand. I've pulled master twice before making any changes...

Reply:
I don't know, you edited those. Maybe restore this how it was before?

Reply:
I figured this out. The problem caused by extension which automatically updates table of contents based on headings. Fixed